documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+      4. Advising other competition officials when necessary.
+      5. Deciding about disqualifying competitors during the competition.
+      6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
+      7. Making a copy of the WCA Regulations available at the competition.
+   2. The WCA Delegate may delegate some of their responsibilities to organizers or other officials.
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The name must not be too general. It should be related in some way to the competition itself, such as:
+      1. The location of the competition
+      2. The competition venue
+      3. The season or date of the competition
+      4. The events of the competition
+      5. The convention the competition is a part of
+      6. A local cultural reference
+      7. A sponsor of the competition
+   2. Capitalization of the competition name must be appropriate. The first letter of each word in the competition’s name should be capitalized and subsequent letters within words should be lowercase, except articles and prepositions, which should be always lowercase. Acronyms should be capitalized.
+   3. The name must end with the year of the last day of the competition.
+   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, and 0-9, with or without accents. Only '.:-& are permitted as special characters.
+   5. The competition name must be 50 characters or less, including whitespaces.
+   6. If the competition is named after anyone or anything, they must be persons or things lacking any controversial meaning worldwide.
+   7. Names should contain the location of the competition.
+   8. Competitions may not be named after a living cuber, competition organizer, or any competition staff.
+   9. Names must not contain possibly offensive parts or memes.
+   10. The name cannot contain website domains or anything susceptible to be interpreted as a link, hashtag or in general an electronic identifier.
+   11. The name cannot contain any trademarks (such as "Nike") unless the organization team is formally authorized, in writing, to use such names.
+   12. Names should not contain any unnecessary parts in the name, like "competition".
+   13. The competition nickname and ID must resemble the competition name.
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
```

Comment:
```suggestion
       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
    14. Abbreviations and Acronyms should be avoided unless the competition name is too long (more than 32 characters).
```
This would also serve as "too generic", but I feel the extra clarification is good. Example: BU 2019 would not be accepted.

Reply:
Agreed.

Reply:
Seeing as this seems to be standard practice for WCAT it would be beneficial to have it written down somewhere.

Perhaps we should add another exception to the competition ID rule so that it may contain the acronym version to improve searchability on the WCA website - e.g. Please Be Quiet Oshawa 2019 can easily be found if I type in "PBQ Oshawa" into the search box, but searching for "PBQ Korea" gives nothing.

Reply:
I’d disagree with that. If we want to increase searcability, we should add functionality to the search feature itself. 
A competition should only have 1 name. Right now there are 3 boxes used for the competition name, allowing for 3 different names to be used.
If we want to increase search ability, why stop at 3 names, why not give the competition 100 different searchable names?

Reply:
I agree that a competition should have one name. But teaching a search function a thousand different acronyms is hard. Having an ID reflect a shorter version of that one competition name is easier. I agree we should avoid acronyms in official names where possible, but we should recognise that acronyms and shortenings do exist for convenience (even when the full text is less that 32 characters).

For example, Melbourne had St Kevin's College Spring 2018, but everyone called it "SKC Spring". If someone asks "are you going to register for SKC?" We should make it as easy as possible for that competitor to find the competition their friend is talking about.

Reply:
Hello Ed, I'll summarize a conversation with the UK delegates here:
- You are asking for a search capability, not something the ID of a competition should stand for. The ID is a unique field used in the urls and data identifiers for the competition. It is just the name of the competition without spaces. If you want to increase search capabilities that's something different that should be implemented elsewhere. It certainly doesn't require writing thousands of patterns of code to produce. Another idea would be to add searchable tags for a competition (similar to Youtube tags) where if you search for those keywords, it will pop up in the search box. Again this is a different feature unrelated to the id.
- By having the ID be different, we are essentially creating multiple names for a singular competition. Why stop at just 3 (name, nickname, and ID), why can't I have 100 names for a competition? the idea here is to make sure that there are as few officially recognized names as possible for a competition (ideally 1, but 2 if it is over 32 characters).
- Shortening the ID to an acronym heightens the possibility of duplicates among them. (IE China Championship 2019 and Classic City 2019 would want the same CC2019 id). Why would it be fair to let one competition have this over another. (of note: yet, this can be an issue for strait up claiming a name. But from what I've seen delegates have been rather respectful and not taking away names from one another).
- We are wanting more worldwide consistency. So If we have one competition have the name be an acronym, we should probably have them all be like this. You run into the issue above and this just becomes a mess.
- Readability becomes an issue as SKC2019, we have no idea what this competition is for or even what competition name that it relates to. Overall, this just makes a list of Ids a list of nonsense: Ags2014, SCMU2015, SIGI2015, GBCO2015, PoP2015, GIC2016, OFFA2016, SCMU2016, ABHC2016 theres about 50 names like these in the database right now which you cant really make sense of.

Reply:
Thanks for the explanation Jacob. I agree that the best solution is probably tags or similar in the website. I'm certainly not supportive of multiple names for a competition - there should be one official name. But there should also be scope to allow people to easily find the competition when they know it as only a simplification of the official name. Currently the easiest way to do that is through the competition ID. Given ambiguity, the potential for duplicates etc. we should use tags or similar instead as a long term solution. I feel in the mean time WCAT should perhaps be more generous with allowing different IDs in some cases when they contain well known acronyms that can ease search ability (e.g. ABHC, PBQ).

Reply:
Has an issue about this feature requests been open in https://github.com/thewca/worldcubeassociation.org?

Related to the suggestion: I'm editing it to change the `A` of `Acronyms` to a lowercase `a` and commiting it.